doc/languages-frameworks/factor.section.md

```diff
@@ -0,0 +1,191 @@
+# Factor {#sec-language-factor}
+
+## Development Environment {#ssec-dev-env}
```

Comment:
```suggestion
## Development Environment {#ssec-factor-dev-env}
```

I strongly recommend namespacing those identifiers, because contributors do use them for mnemonics, and they will be quite confused if `dev-env` is not about generic development environments.

Reply:
Sure!